### PR TITLE
tweak custom admin page docs

### PR DESCRIPTION
**Pull request by raayu83**
Implementation of https://github.com/unfoldadmin/django-unfold/issues/641

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -734,35 +734,41 @@ class YourModelAdmin(ModelAdmin):
...
+@admin.register(MyModel)
 class CustomAdmin(ModelAdmin):
     def get_urls(self):
-        return super().get_urls() + [
+        return [
             path(
                 "custom-url-path",
                 MyClassBasedView.as_view(model_admin=self),  # IMPORTANT: model_admin is required
-                name="custom_name"
+                name="custom_name",
```
Comment (krystianmagdziarz) [inline README.md:762]: why that?

**Comment by lukasvinclav** [comment]
@raayu83 why you changed get_urls() method? What was the issue?

**Comment by lukasvinclav** [comment]
Thanks for contribution. Meanwhile, I'm improving the documentation so I manually placed your code into new one.

**Comment by bahattincinic** [comment]
Hi @lukasvinclav  

Screenshot 2024-09-04 at 21 32 17

this documentation is still invalid. 

it should be 

```python
def get_urls(self):
        return  [
            path(
                "custom-url-path",
                MyClassBasedView.as_view(model_admin=self),  # IMPORTANT: model_admin is required
                name="custom_name"
            ),
        ] + super().get_urls()
```

**Pull request closed without merging**